### ISSUE TITLE
CC conditional colouring

### ISSUE THREAD

**Issue by kylelt** [plotly/dash-table-experiments#7]
Not sure if this is within scope of this project but being able to write methods to conditionally colour rows would be fantastic

**Comment by chriddyp** [comment]
Good idea @kylelt . Would you like to color rows by a certain numerical value (like a heatmap) or just customize the colors with regular color values (e.g. to make a striped table)

**Comment by kylelt** [comment]
Heatmap !

**Comment by Manuel3112** [comment]
Conditional coloring of cells (depending on numeric or string value) would be very useful indeed...

**Comment by chriddyp** [comment]
This has been done in the non-experimental version of this project https://github.com/plotly/dash-table/. See https://dash.plot.ly/datatable/style

**Comment by Manuel3112** [comment]
saw that thx! do you know when it will be implemented in the experimental version?

**Comment by chriddyp** [comment]
It won't be. We're no longer working on this experimental version. It was just a place for us to work on some prototypes before committing to a particular implementation.

**Comment by Manuel3112** [comment]
Ok, thx for the info!
